server/src/main/java/org/elasticsearch/common/network/NetworkService.java

```diff
@@ -51,9 +51,9 @@
     public static final Setting<Boolean> TCP_KEEP_ALIVE =
         Setting.boolSetting("network.tcp.keep_alive", true, Property.NodeScope);
     public static final Setting<Integer> TCP_KEEP_IDLE =
-        Setting.intSetting("network.tcp.keep_idle", -1, -1, Property.NodeScope);
+        Setting.intSetting("network.tcp.keep_idle", -1, -1, 300, Property.NodeScope);
```

Comment:
This bit needs a breaking change tag (and docs in the backport)

Reply:
ok, done so